modules/configuring-egress-proxy-edns-operator.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * networking/external_dns_operator/nw-configuring-cluster-wide-egress-proxy.adoc
+
+:_content-type: PROCEDURE
+[id="nw-configuring-cluster-wide-proxy_{context}"]
+= Configuring the External DNS Operator to trust the certificate authority of the cluster-wide proxy
+
+. Create the config map to contain the CA bundle in the `external-dns-operator` namespace by running the following command:
```

Comment:
This missing a module introduction and the `.Procedure` heading.

Reply:
Addressed